frontend/src/components/searchSelector/SearchSelector.tsx

```diff
@@ -39,7 +39,7 @@ type SearchSelectorProps = {
   searchHelpText?: string;
   searchPlaceholder?: string;
   searchValue: string;
-  toggleText: string;
+  toggleText: React.ReactNode | string;
```

Comment:
am i going crazy here? i swear i reviewed this piece of code yesterday in a different PR and said maybe this should be `toggleContent` now that it is not just text 😆 

but i can't find it 🤔

Reply:
sorry, this needs a rebase. This was addressed in my previous PR https://github.com/opendatahub-io/odh-dashboard/pull/3990. It needs to be merged. Once it is, I will rebase this.

Reply:
oh, here https://github.com/opendatahub-io/odh-dashboard/pull/3990/files#diff-d02f69cf99d2a9d940792401f88c96df38f300c012cd5804761be88342ced8de